edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/MetricTransformer.cs

```diff
@@ -30,13 +31,21 @@ public IEnumerable<Metric> FilterMetrics(IEnumerable<Metric> metrics)
                     continue;
```

Comment:
So allowed tags will bypass transformation? What is the reason for naming this "allowed"? What about "tagsIndicatingNoTransform"

Reply:
nope, this acts as a whitelist. Only metrics that contain the given tags will be allowed through, all others are dropped. It used to be called whitelist, but was renamed.